instrumentation/couchbase/couchbase-2-common/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/couchbase/v2_0/CouchbaseRequestInfoHolder.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.couchbase.v2_0;
+
+import static io.opentelemetry.context.ContextKey.named;
+
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.ContextKey;
+import javax.annotation.Nullable;
+
+public final class CouchbaseRequestInfoHolder {
```

Comment:
the name doesn't seem right anymore now that it's not exactly a "holder", maybe just move the static stuff into `CouchbaseRequestInfo`? it looks like we have that pattern a couple other places, e.g. `AppServerBridge`, `ServletContextPath`, `GrizzlyErrorHolder`